src/tenderly/spawn-devnet.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-var-requires, no-console */
+import fs from "fs";
+import dotenv from "dotenv";
+import util from "util";
+import proc from "child_process";
+
+const asyncExec = util.promisify(proc.exec);
+
+dotenv.config();
+
+const {
+  TENDERLY_ACCESS_KEY,
+  TENDERLY_PROJECT_SLUG,
+  TENDERLY_ACCOUNT_ID,
+  TENDERLY_DEVNET_TEMPLATE,
+} = process.env;
+
+// eslint-disable-next-line max-len
+const command = `tenderly devnet spawn-rpc --project ${TENDERLY_PROJECT_SLUG} --template ${TENDERLY_DEVNET_TEMPLATE} --account ${TENDERLY_ACCOUNT_ID}  --access_key ${TENDERLY_ACCESS_KEY}`;
+
+
+const spawnDevNet = async () => {
+  const { stderr } = await asyncExec(command);
+  const devNetUrl = stderr.trim().toString();
+
+  console.log(`DEVNET_RPC_URL=${ devNetUrl }`);
+
+  // if file doesn't exist, create it
+  if (!fs.existsSync(".env")) {
```

Comment:
just add this to the options for the `writeFileSync` call below, it will append to an existing file or create a new file if one doesn't exist automatically using the `w` flag 

`fs.writeFileSync(".env", { flag : "w" });`

Reply:
actually, I think "w" is already there by default. will check if it works

Reply:
Please confirm, I've seen a few posts on stackoverflow that it is the default according to the docs but it doesn't behave that way